test/suites/snapshots.sh

```diff
@@ -7,35 +7,59 @@ test_snapshots() {
   lxc init testimage foo
 
   lxc snapshot foo
-  [ -d "${LXD_DIR}/snapshots/foo/snap0" ]
+  # FIXME: make this backend agnostic
+  if [ "${LXD_BACKEND}" = "dir" ]; then
+    [ -d "${LXD_DIR}/snapshots/foo/snap0" ]
+  fi
 
   lxc snapshot foo
-  [ -d "${LXD_DIR}/snapshots/foo/snap1" ]
+  # FIXME: make this backend agnostic
+  if [ "${LXD_BACKEND}" = "dir" ]; then
+    [ -d "${LXD_DIR}/snapshots/foo/snap1" ]
+  fi
 
   lxc snapshot foo tester
-  [ -d "${LXD_DIR}/snapshots/foo/tester" ]
```

Comment:
No, it doesn't with zfs. zfs doesn't make this directory until you actually start the snapshot (viz. my fix above)